packages/react-theme/package.json

```diff
@@ -16,12 +16,14 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
+    "start": "echo \"This is DEPRECATED instead use 'storybook'\" && just-scripts dev:storybook",
+    "storybook": "start-storybook",
     "start-test": "just-scripts jest-watch",
     "test": "just-scripts test",
     "update-snapshots": "just-scripts jest -u"
   },
   "devDependencies": {
+    "@ctrl/tinycolor": "^3.3.4",
```

Comment:
This is an external dependency used in storybook only.
How should that be added so that
1. import/no-extraneous-dependencies is happy
2. package bundle size is not affected
3. deployed suite storybook works

Reply:
Having this as a devDependency does not work - lint complains
```
fluentui/packages/react-theme/src/_private/ColorRamp.tsx
  9:27  error  '@ctrl/tinycolor' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
```
I did not manage to suppress the error, so I removed the dependency and hope react-examples depends on it and the dependency is hoisted to the root 👎

Reply:
You can add the relevant path to `devDependenciesFiles` in `packages/eslint-plugin/src/utils/configHelpers.js`.

Reply:
Moved to root